src/ResourceManager/ServiceBus/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 -->
 ## Current Release
 
+* Bug fix : some properties of Queue object were not updated beacuse of the space inserted in there name in swagger spec. 
```

Comment:
Thin k of this form the customer perspective - what would the customer need to know about this - just what the affects of the bug were and that it was fixed, not the root cause.